### PR TITLE
Re-add Lwan to the suite

### PR DESCRIPTION
**Pull request by lpereira**
Source code for the benchmark is now included in this repository. Since
there's no recent version of Lwan, however, a specific Git hash is
obtained from the repository so that the framework can be built.

At the moment, only the JSON and Plaintext tests are enabled; the others,
including Fortunes, DB, and Queries, have bitrotted and need to be
adapted to conform to current rules.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ env:
   matrix:
     - "TESTLANG=C"
     - "TESTLANG=CSharp"
+    - 'TESTDIR="C/lwan"'
```
Comment (nbrady-techempower) [inline .travis.yml]: You can remove this line. The framework will be tested with `"TESTLANG=C"`

Comment (lpereira) [reply .travis.yml]: Done.

File: frameworks/C/lwan/lwan.dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+RUN apt update -yqq
+RUN apt install -yqq \
+	git pkg-config build-essential cmake zlib1g-dev \
+	libsqlite3-dev libmariadbclient-dev wget
+
+ADD ./ /lwan
+WORKDIR /lwan
+
+RUN git clone https://github.com/lpereira/lwan --depth 1 && \
```
Comment (volyrique) [inline frameworks/C/lwan/lwan.dockerfile]: A shallow clone looks quite fragile, and I think that you can achieve the same effect simply by downloading `https://github.com/lpereira/lwan/archive/${commit hash}.zip`.

Comment (lpereira) [reply frameworks/C/lwan/lwan.dockerfile]: Thanks.  That's probably the reason it's failing in CI.  I've just pushed a new version using your suggestion; seems to be working locally.

**Pull request merged**